tests/rptest/services/redpanda.py

```diff
@@ -734,6 +734,11 @@ class RedpandaServiceBase(Service):
         'kafka_connections_max': 2048,
         'kafka_connections_max_per_ip': 1024,
         'kafka_connections_max_overrides': ["1.2.3.4:5"],
+
+        # TODO: we need this to not wait too long before moves are scheduled
+        # after decommission. Get rid of this after event-driven balancer execution
+        # is implemented.
+        "partition_autobalancing_tick_interval_ms": 5000,
```

Comment:
I was wondering how most decomm tests passed :-D.. my other patch is ready.. it conflicts with this patch.. once this is merged, I can rebase on top.